context/app/static/js/components/Search/Search.scss

```diff
@@ -28,8 +32,8 @@
     }
     td a {
         display: block;
-        margin: -10em;
-        padding: 10em;
+        margin: -100%;
+        padding: 100%;
```

Comment:
Yeah: Something copied from SO. Not attached to this approach.
```
    // Force <a> to fill each cell, so the whole row is clickable.
    // https://stackoverflow.com/questions/3966027
```
(Changed because 10em was not sufficient when the cell was wide but the text was short.)